README.md

```diff
@@ -31,14 +31,28 @@ allprojects {
 
 Add the project as a dependency:
 
-```java
+```
 dependencies {
 	compile 'com.github.stirante:lol-client-java-api:1.2.3'
 }
 ```
 
 ### Maven
 
+
+#### 1. Nexus Repo
```

Comment:
This section also needs XML for referencing Nexus Repo

Reply:
(are you sure? I was never required to add the repo for the external project to work, anyways...) I rolled back the change and just bumped the version of the SNAPSHOT in the README